src/track/track.h

```diff
@@ -327,6 +327,8 @@ class Track : public QObject {
     // GlobalTrackCacheResolver!
     void initId(TrackId id); // write-once
```

Comment:
write-once is no longer true

initId() and resetId() and share the same restrictions mentioned in the comment above!